infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -346,7 +347,18 @@ static void restartServer(boolean resetData) {
     private static void cleanShutdown() {
         log.info("CleanShutdown: Starting Cleanup.");
         shutdownServer = true;
-        activeServer.close();
+        try {
+            CorfuServerNode current = activeServer;
+            if (current != null) {
+                activeServer.close();
```

Comment:
it has to be `current ` instead of `activeServer `

Reply:
Yep.